package.json

```diff
@@ -18,7 +18,7 @@
     "expo": "^35.0.0",
     "expo-barcode-scanner": "~7.0.0",
     "expo-camera": "^7.0.0",
-    "expo-constants": "^7.0.1",
+    "expo-constants": "~7.0.0",
```

Comment:
What's the rationale behind this change?

Reply:
This was probably because we didn't know expo-constants was installed so ran `expo install` instead of npm, `expo install` grabs the correct version for the sdk from npm which explains the downgrade and constant version! should I discard this change?